charts/openstack-cluster/templates/_helpers.tpl

```diff
@@ -203,13 +203,15 @@ files:
       # This file is created by the capi-helm-chart to ensure that its parent directory exists
     owner: root:root
     permissions: "0644"
+{{- if not (.Values.ignitionBasedOS | default false) }}
```

Comment:
I would prefer we just set `ignitionBasedOS: false` in the chart `values.yaml`, then we don't need the `default` here.

Reply:
Similarly elsewhere